tt_metal/hw/ckernels/grayskull/metal/llk_api/llk_sfpu/ckernel_sfpu_softplus.h

```diff
@@ -0,0 +1,47 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "ckernel.h"
+#include "ckernel_defs.h"
+#include "ckernel_sfpu_converter.h"
+#include "ckernel_sfpu_exp.h"
+#include "sfpu/ckernel_sfpu_log.h"
+
+using namespace sfpi;
+
+namespace ckernel {
+namespace sfpu {
+
+template <bool APPROXIMATION_MODE, int ITERATIONS = 4>
+sfpi_inline void calculate_softplus(uint param0, uint param1, uint param2) {
+    vFloat beta = Converter::to_float(param0);
+    vFloat beta_reciprocal = Converter::to_float(param1);
+    vFloat threshold = Converter::to_float(param2);
+    for (int d = 0; d < ITERATIONS; d++) {
+        vFloat a = dst_reg[0] * beta;
+        v_if(a >= threshold) {
+            dst_reg++;
+            return;
```

Comment:
The return; needs to be replaced with continue;

Reply:
No longer relevant, since GS will not be supported